pkg/builder/auth_test.go

```diff
@@ -73,3 +80,57 @@ var _ = DescribeTable("IsPrivilegedAccount",
 		true,
 	),
 )
+
+var _ = Describe("VerifyWebhookRequest", func() {
+	BeforeEach(func() {
+		dir := filepath.Dir(caFilePath)
+		err := os.MkdirAll(dir, 0755)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
Here and below - can combine these two lines into a single `Expect(os.MkdirAll(dir, 0755)).To(Succeed())` statement.